src/Cohesity.Powershell/Scripts/Utility/CSLog.ps1

```diff
@@ -0,0 +1,31 @@
+function CSLog
+{
+    param (
+        [Parameter(Mandatory)]
+        [string]$Message,
+        
+        [Parameter()]
+        [ValidateSet('1','2','3')]
+        [int]$Severity = 1 ## Default to a low severity. Otherwise, override
+    )
+    
+    $line = [pscustomobject]@{
+        'DateTime' = (Get-Date)
+        'Message' = $Message
+        'Severity' = $Severity
+    }
+    $logFileName = "cohesityPS.log"
+    if($null -eq $CSLogFilePath) {
+        $CSLogFilePath = "$HOME/" + $logFileName
```

Comment:
can we check if it can use any of existing variables:
```
Get-CohesityPrivilege -
 Debug                WarningAction        ErrorVariable        InformationVariable  OutBuffer
Verbose              ErrorAction          InformationAction    WarningVariable      OutVariable          PipelineVariable
```

I believe we store the accessToken variable in $HOME/.cohesity , we need leverage and write logs in same path or make this a folder structure so that we have 

cohesity/
    profiles
    cmdlet.log
    currentaccessToken.json
Something on above lines . I used aws commandlets, they had some thing like that.

Reply:
Did not get the context on using Get-CohesityPrivilege
Need clarity for currentaccessToken.json.
Fixed : Amended the changes by appending a folder name $HOME/cohesity for the logs.
            Changed the log file name to cmdlet.log

Reply:
A cohesity cmdlet configuration mechanism has been implemented.
Set-CohesityCmdletConfig  and Get-CohesityCmdletConfig

No clarity how currentaccessToken.json could help us, and how profiles will be useful.

Reply:
Cool .. sounds good. 

regarding currentaccessToken , I was thinking if we have profiles , we could take the latest profile's access token information from this. 
Let me know if thats ok.

Reply:
Lets discuss and finalize the design

Reply:
Will create a separate ticket and track the feature implementation of user profile, that would also contain currentaccessToken